Proposals/0001-bas-broek.md

```diff
@@ -0,0 +1,49 @@
+# Scaling A Codeless Open Source Swift Community with Bas Broek
+
+* Proposal: [will add PR#](will add PR link)
+
+* Authors: [Garric Nahapetian](https://github.com//garricn)
+
+* Review Manager: [will add name](will add profile link)
+
+* Status: Proposed
+
+## Introduction
+
+Garric, Chris, and Bas will use Bas's experience as the steward of Swift Weekly Brief as a jumping off point to discuss the challenges and lessons learns with scaling a mostly code-less open source Swift community.
+
+## Motivation
+
+Brief description of your motivation behind this episode
+
+The Swift Community Podcast is an experiment and is in its infancy and one of the biggest challenges is how to effectively scale the project through adequate community engagement. A codeless open source community may present challenges that are distant from code centric ones and so Bas maybe have some valuable insight into this given his experience with shepherding the Swift Weekly Brief.
```

Comment:
I think it is also part of the motivation to  increase collaboration and drive contributors to the Brief as well.

Reply:
and vice versa!